File: temp_add_location.py
from pathlib import Path

path = Path('app/page.tsx')
text = path.read_text(encoding='utf-8')
old = """  {
    label: 'Email',
    href: 'mailto:ahmedch8990@gmail.com',
  },
  {
    label: 'Phone',
    href: 'tel:+923411003874',
  },
];
"""
new = """  {
    label: 'Email',
    href: 'mailto:ahmedch8990@gmail.com',
  },
  {
    label: 'Phone',
    href: 'tel:+923411003874',
  },
  {
    label: 'Location',
    href: 'https://www.google.com/maps/search/Pakistan',
  },
];
"""
if old not in text:
    raise SystemExit('Pattern not found')
path.write_text(text.replace(old, new), encoding='utf-8')
print('updated')

File: temp_add_location.js
const fs = require('fs');
const path = require('path');
const filePath = path.join(__dirname, 'app', 'page.tsx');
let text = fs.readFileSync(filePath, 'utf8');
const oldPart = `  {
    label: 'Email',
    href: 'mailto:ahmedch8990@gmail.com',
  },
  {
    label: 'Phone',
    href: 'tel:+923411003874',
  },
];
`;
const newPart = `  {
    label: 'Email',
    href: 'mailto:ahmedch8990@gmail.com',
  },
  {
    label: 'Phone',
    href: 'tel:+923411003874',
  },
  {
    label: 'Location',
    href: 'https://www.google.com/maps/search/Pakistan',
  },
];
`;
if (!text.includes(oldPart)) {
  throw new Error('Pattern not found');
}
text = text.replace(oldPart, newPart);
fs.writeFileSync(filePath, text, 'utf8');
console.log('updated');
